.ls-lint.yml

```diff
@@ -0,0 +1,15 @@
+ls:
+    '{cypress,scripts,src,types}':
+        .dir: kebab-case
+        .ts: kebab-case
+        .test.ts: kebab-case
+        .tsx: kebab-case
+        .test.tsx: kebab-case
+        .d.ts: kebab-case
+        .cy.ts: kebab-case
+        .module.css: kebab-case
+        .sh: kebab-case
+        .json: kebab-case
```

Comment:
IMO all files and folders should use the same casing. I like kebab-case, but happy to change to something else, as long as it's consistent